.evergreen/functions.yml

```diff
@@ -40,11 +40,7 @@ variables:
     EVERGREEN_VERSION_ID: ${version_id}
     EVERGREEN_WORKDIR: ${workdir}
     EVERGREEN_CREATED_AT: ${created_at}
-    # WARN: This version is behind our electron runtime, but updating it will
-    # drop support for some older linux platforms, so we are keeping them out of
-    # sync for now
-    # TODO: https://jira.mongodb.org/browse/COMPASS-6915
-    NODE_JS_VERSION: '18.19.1'
+    NODE_JS_VERSION: '20.16.0'
     NPM_VERSION: '10.2.4'
```

Comment:
Don't we want to bump the npm version to be the one that comes with node 20 so that when someone installs or uses node 20 locally they don't have to know or remember to also change npm?

Reply:
Nevermind. Unchanged in the required fields in package.json. Ignore me.